node_scripts/install/spark.py

```diff
@@ -88,12 +88,14 @@ def start_history_server():
     # configure the history server
     spark_event_log_enabled_key = 'spark.eventLog.enabled'
     spark_event_log_directory_key = 'spark.eventLog.dir'
+    spark_history_fs_log_directory = 'spark.history.fs.logDirectory'
```

Comment:
You should probably add this as a commented out field at https://github.com/Azure/aztk/blob/master/aztk_cli/config/spark-defaults.conf